packages/ses/src/lockdown-shim.js

```diff
@@ -35,6 +33,19 @@ function assert(condition, message) {
   }
```

Comment:
Why not import this from `assert.js`? It is currently identical, and it will make it easy to upgrade all occurrences to the new assert system when it is time to do that.